app/partials/states/admin_events.html

```diff
@@ -183,3 +184,30 @@ <h4>User search results ({{userSearch.results.length}})</h4>
     </div>
 </div>
 
+<div isaac-modal="eventBookingModal">
+  <div class="reveal-modal-head">
+    <h2>Admin - Create a <span ng-show="userBookings.indexOf(result._id) == -1 && eventSelected.placesAvailable <= 0">waiting list</span> booking</h2>
+    <a class="close-reveal-modal" href="javascript:void(0)"><span class="hide-for-small-only">Close</span> <span class="icon-close"></span></a>
+  </div>
+  <div class="reveal-modal-body">
+        <div event-booking-form></div>
```

Comment:
This ends up adding two booking form modals to the DOM, which leads to a browser warning about repeated IDs. I can't really work out how though . . .

Reply:
This doesn't seem to cause any ill effects and as it is only visible on the admin view I think we should accept it. The isaac-modal.js appears to clone the whole dom for a modal hence the error. We could fix this in the future if it causes problems.